### ISSUE TITLE
Feature: Add keywords (Given, When, Then) to html report

### ISSUE THREAD

**Issue by vitalets** [vitalets/playwright-bdd#69]
**The problem**
Currently html-report does not contain keywords (Given, When, Then):

<img width="50%" src="https://github.com/vitalets/playwright-bdd/assets/1473072/6975fb2d-e3f5-4154-8a1a-a80c206fa6af" />

It should be:
```
Given I am on Playwright home page
When I click link "Get started"
Then I see in title "Installation"
```

**A solution**
Pass keywords to Playwright steps to display them in report.
Use exactly the same keywords as used in feature file (e.g. don't convert And -> Given).
Handle i18n keywords.

Keep in mind Cucumber reporters: #9

**Comment by meeranharish** [comment]
@vitalets 
Hi, 
You have proposed an work around to pass keywords to the steps.  Can we then use the steps in different context ?

Eg:
Given User logs out
When User login
Then User access main menu

Eg2:
Given User login
When User logs out
Then User cannot access main menu

In these examples we used the same step but in different context 'given' in eg1 and 'when' in eg 2
Can we use same step implementation for them if we pass the keyword to the step definition name?

Or did i misunderstood you?

**Comment by vitalets** [comment]
Hi @meeranharish 
Yes, scenarios are relax on keywords in step definitions, e.g. step defined with `Given` can be used also with `When` or `Then`.
The purpose of this issue is to display steps in html-report with corresponding keyword. For your example report should show exactly strings that you typed in scenario:
```
Given User logs out
When User login
Then User access main menu

Given User login
When User logs out
Then User cannot access main menu
```
Instead of current behavior:
```
User logs out
User login
User access main menu

User login
User logs out
User cannot access main menu
```

**Comment by meeranharish** [comment]
Thanks for the response. Yes, the steps are not tied to keywords but only that we might miss the keyword in the report. 
As you proposed this
"
A solution
Pass keywords to Playwright steps to display them in report.
"
I felt that might bring in the restriction for the sake of reporting, right?

**Comment by vitalets** [comment]
> I felt that might bring in the restriction for the sake of reporting, right?

Got it. No, steps will not be restricted to keywords.

**Comment by meeranharish** [comment]
@vitalets 
whats your suggested way to pass the keywords to the step?

**Comment by vitalets** [comment]
> whats your suggested way to pass the keywords to the step?

This is just internal implementation details. Nothing will change from the usage point.
Currently we actually pass keyword to the step - via `Given / When / Then` calls in generated files:
```ts
    await Given("State 1");
    await When("Action 1");
    await Then("Assertion 1");
```

**Comment by meeranharish** [comment]
> > whats your suggested way to pass the keywords to the step?
> 
> This is just internal implementation details. Nothing will change from the usage point. Currently we actually pass keyword to the step - via `Given / When / Then` calls in generated files:
> 
> ```ts
>     await Given("State 1");
>     await When("Action 1");
>     await Then("Assertion 1");
> ```

Is this sufficient enough to get the corresponding keywords in the html report for each step? Currently we don't get right?

**Comment by meeranharish** [comment]
Looking forward :)

**Comment by vitalets** [comment]
Released in 5.5.0 for English keywords.
Now html report is the following:
<img width="80%" src="https://github.com/vitalets/playwright-bdd/assets/1473072/e9ecd236-91f7-4f2f-bfd5-f6d1c1821f42" />
